index.js

```diff
@@ -17,11 +21,20 @@ module.exports = function (source) {
 		return;
 	}
 
-	var compiler = new KaitaiStructCompiler();
-	var kaitaiImport = 'var KaitaiStream = require("kaitai-struct/KaitaiStream")\n';
+	const moduleDir = this.context;
+
+	var yamlImporter = {
+		importYaml: function(name, mode) {
+			const filePath = path.join(context, name + ".ksy");
```

Comment:
```suggestion
			const filePath = path.join(moduleDir, name + ".ksy");
```

Reply:
Oh man, I'm really sorry for being so sloppy 😊